server.py

```diff
@@ -0,0 +1,352 @@
+import flask
+from flask import Flask, json
+import waitress
+
+from datetime import datetime, timedelta
+
+from core import Controller
+from models import FoodCategory, FoodDetails, CustomerDetails, CustOrderSelection, CustOrderStatus
+
+
+not_found = {"message": "Not Found"}
+success = {"message": "Success"}
+failed = {"message": "Failed"}
+
+ 
+def create_app(fos):
+    """
+    Creates the server app.
+    """
+    app = Flask('Food Ordering System')
+
+    ### employee routes
+    @app.route('/employee/add_food_category', methods=['POST'])
```

Comment:
The PATH of a REST API should not have "underscore" convention. It's usually "hyphen" convention.

So instead of doing `/employee/add_food_category`, use hyphen `/employee/add-food-category`.

Change the PATH for all the URLs and also the sample requests in the comments in the function definition.

Reply:
changed PATH